IO.Eventuate.Tram.Tests/TestHelpers/TestServicesHost.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using IO.Eventuate.Tram.Local.Kafka.Consumer;
+using IO.Eventuate.Tram.Database;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Hosting;
+
+namespace IO.Eventuate.Tram.Tests.TestHelpers
+{
+    public class TestServicesHost
```

Comment:
MessageTest and EventTest need different configuration.
Each one should have their own service host.
The common code should be in an abstract super class

Reply:
Added Separate Service host in Unit tests itself